changelogs/unreleased/3787-key-error-export.yml

```diff
@@ -0,0 +1,3 @@
+description: Adds a human readable error on key error when exporting
```

Comment:
This is no longer accurate: we just cleanly wrap any exception that occurs during resource export so that it is more useful to the end user.
Additionally, I think a `bugfix` section entry would be appropriate here.